boxoffice/templates/order_refund_mail.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset='UTF-8'>  
+  <title>Order refund</title>
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:400italic,600italic,400,600" />
+  <style type="text/css">
+    * { 
+      margin: 0; 
+      padding: 0; 
+    }
+
+    body {
+      font-family: "Source Sans Pro",Arial,sans-serif;
+      font-size: 16px;
+      line-height: 1.2;
+      color: #333;
+      background-color: #fff
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+    p { 
+      margin: 5px 0 10px;
+    }
+
+    .terms {
+      margin-top: 20px;
+    }
+  </style>
+</head>
+
+<body>
+  <div>
+
+    <p>Hello {{ order.buyer_fullname }},</p>
+
+    <p>
+      A refund of {{ currency_symbol }} {{ refund_amount }} has been initiated against your order (Receipt No: {{ order.invoice_no }}). You should receive the refund in 2-5 days.</p>
```

Comment:
Why is the user getting a refund? It would help if the admin who is issuing a refund, can leave two notes:

1. An internal note that explains why this refund was issued.
2. A note to the user to explain why they're getting a refund.